### PR TITLE
WildCam Darien Map part 7: Regions

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview
* This PR adds the 'region' layers to the map.
* Notably: the park boundaries for the two national parks, and the areas of vegetation.
screen shot 2017-08-29 at 16 11 38
* These layers can be controlled by the 'Layers' control on the top-left corner of the Map.

### Potential for Improvement
* This map has no legends. It could probably use some?
* The layer order is fairly random? So sometimes the Data (camera map markers) layer is below the park boundaries, which isn't what we want? 

### Status
Ready for review, but it'll make more sense to merge PR #39 and #40 first, then review & rebase this branch onto the new master.

### CODE REVIEW

**Comment by shaunanoordin** [comment]
## PR Update
* This PR now needs to be rebased. I'll update when PR #40 has been OK-ed.
* Anyways, new content: a very simple Map Legend has been added.
screen shot 2017-08-29 at 21 29 52

File: src/components/maps/SimpleMapLegend.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from 'grommet/components/Box';
+
+const SimpleMapLegend = (props) => {
+  return (
+    <Box className="map-legend-simple">
```
Comment (srallen) [inline src/components/maps/SimpleMapLegend.jsx]: Maybe use a Grommet List instead?

File: src/components/maps/SimpleMapLegend.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      {Object.keys(props.items).map(key => {
+        const value = props.items[key];
+        return (
+          <Box className="map-legend-item" key={`map-legend-item-${key}`} direction="row">
```
Comment (srallen) [inline src/components/maps/SimpleMapLegend.jsx]: Perhaps ListItem instead?

File: src/components/maps/SimpleMapLegend.jsx (hunk 1 above)
Comment (shaunanoordin) [reply src/components/maps/SimpleMapLegend.jsx]: 👍

File: src/components/maps/SimpleMapLegend.jsx (hunk 2 above)
Comment (shaunanoordin) [reply src/components/maps/SimpleMapLegend.jsx]: 👌

**Comment by shaunanoordin** [comment]
Thanks again for the review, @srallen ! OK, I'm going to upload one more update in an hour:
1. MultiChoiceFilter will be a functional component.
2. Simple Map Legend will go the way of the List.
3. Translations for the Map Layer (+ Map Legend) will be implemented. (See PR #40)

I'll also rebase and force-update this branch once PR #40 is merged. 👍

**Comment by srallen** [comment]
@shaunanoordin I just merged #40, so this is ready for a rebase.

**Comment by shaunanoordin** [comment]
## PR Update
* This PR has been rebased, woot!
* Feedback re: coding has been implemented.
* Additional translations have been added, notably for the map layers.
* Certain map layers have been renamed, e.g. the "Water" layer -> "Water Region" layer because otherwise a super-general word -> translation could screw up the database exports.
* Known bug: The Map Legend doesn't scroll properly on a MacBook touchpad; the scroll works normally with a mouse wheel, but for some reason doing that two-finger scroll causes the map to zoom instead. I'll need to check the event captures/bubbling.

### Status
@srallen , this branch is now ready for (hopefully final!) review. 👍

**Pull request merged**